applications/management/commands/export_applications.py

```diff
@@ -0,0 +1,12 @@
+from typing import Any, Optional
+
+from django.core.management.base import BaseCommand
+
+from ...utils.export_data import ApplicationDataExporter
+
+
+class Command(BaseCommand):
+    help = "Exports application data from the database"
+
+    def handle(self, *args: Any, **options: Any) -> Optional[str]:
```

Comment:
This also should have parametrization by application round

Reply:
Parameterization implemeted to the command.